looprpc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16.3-buster
+FROM golang:1.19.10-buster
```

Comment:
Here's an example for the changes required to the `protobuf-compiler` and `clang-format` versions if you wanted to go to a `-bookworm` (Go 1.2x) version: https://github.com/lightninglabs/taproot-assets/blob/main/taprpc/Dockerfile#L1
It's not too involved, so I would suggest doing it.

Reply:
Fixed